src/components/ux-alert/ux-alert.js

```diff
@@ -0,0 +1,9 @@
+Ractive.extend({
+	template: Ractive.defaults.templates['ux-alert'],
+	oninit: function ( options ) {
+		this.on( 'closeClicked', function () {
+			this.teardown();
+			return false;
```

Comment:
Q) Do we want to prevent the bubbling of this event? Is `.on('teardown')` enough?